buffer/src/main/java/io/netty5/buffer/ByteBufUtil.java

```diff
@@ -217,7 +200,7 @@ public static void copy(AsciiString src, int srcIdx, ByteBuf dst, int length) {
                     buf.append(' ');
                 }
                 BYTEPADDING[i] = buf.toString();
-            }
+        }
```

Comment:
nit: seems like formatting is off. 
```suggestion
            }
```